.github/workflows/python.yml

```diff
@@ -171,6 +171,23 @@ jobs:
         with:
           fetch-depth: 0
           submodules: recursive
+      - name: Install ccache
+        run: brew install ccache
```

Comment:
We don't need this because `ccache` is installed by `brew bundle --file=cpp/Brewfile`.